app/components/NavButtons/index.js

```diff
@@ -21,16 +21,16 @@ class NavButtons extends React.Component {
   }
 
   render() {
-    let { back, forward, historyLength, historyCurrentIndex} = this.props;
-
+    const history = this.props.history;
```

Comment:
Instead of this, we could use `withRouter` to enhance this component with history directly.

Reply:
Fixed.